deps/secure-socket/context.lua

```diff
@@ -33,7 +33,7 @@ do
   local isLibreSSL = V:find('^LibreSSL')
 
   _, _, V = openssl.version(true)
-  local isTLSv1_3 = not isLibreSSL and V > 0x10100000
+  local isTLSv1_3 = not isLibreSSL and V > 0x10101000
```

Comment:
Thats means need openssl 1.1.1, not 1.1.0

Reply:
That seems right, see the [Major changes between OpenSSL 1.1.0i and OpenSSL 1.1.1 section here](https://www.openssl.org/news/openssl-1.1.1-notes.html)

> - Support for TLSv1.3 added (see https://wiki.openssl.org/index.php/TLS1.3 for further important information).